v2/json2/json_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"testing"
 
 	"github.com/gorilla/rpc/v2"
+	"strings"
```

Comment:
Could you run `goimport` here?

Reply:
```bash
$ goimports -d v2/json2/json_test.go | wc -l
0
```
🤔

Reply:
Remove `"strings"`, then run `goimports` again. It should add `"strings"` to the import group above, alongside the rest of the standard library imports.

Reply:
🤦‍♂️ 
Done, thanks.